docker-wrappers/DOMINO/metadata.json

```diff
@@ -0,0 +1,4 @@
+{
+    "dockerName": "domino",
+    "architectures": ["linux/amd64"]
```

Comment:
What dictates whether an image can be built for an architecture? Should that go in the DOMINO readme or in some other documentation?